extensions/wikia/InsightsV2/InsightsController.class.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+class InsightsController extends WikiaSpecialPageController {
+	public function __construct() {
+		parent::__construct( 'Insights', 'insights', true );
+	}
+
+	/**
+	 * The main, initializing function
+	 * @throws MWException
+	 */
+	public function index() {
+		wfProfileIn( __METHOD__ );
+		$this->wg->Out->setPageTitle( wfMessage( 'insights' )->escaped() );
```

Comment:
`setPageTitle` is safe, so no need to escape the message passed to it. Also, you should be able to make use of `$this->getOutput()->setPageTitle( $this->msg( 'insights' )->plain() );`